css/css-sizing/aspect-ratio/block-aspect-ratio-042.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<title>CSS aspect-ratio: The transferred minimum height beats the automatic content-based minimum height</title>
```

Comment:
Here and Line 6 below, I think you mean "transferred _maximum_".

Reply:
Done